src/SFA.DAS.PR.Data.UnitTests/Models/RequestModelTests.cs

```diff
@@ -0,0 +1,41 @@
+﻿using NUnit.Framework;
+using SFA.DAS.PR.Data.UnitTests.Setup;
+using SFA.DAS.PR.Domain.Entities;
+using SFA.DAS.PR.Domain.Models;
+using SFA.DAS.ProviderRelationships.Types.Models;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.PR.Data.UnitTests.Models;
+
+public class RequestModelTests
+{
+    [Test]
+    [RecursiveMoqAutoData]
+    public void RequestModel_ImplicitConversion_Successful()
+    {
+        Request request = RequestTestData.Create(Guid.NewGuid());
+
+        RequestModel requestModel = request;
+
+        Assert.Multiple(() =>
+        {
+            Assert.That(requestModel.RequestId, Is.EqualTo(request.Id));
```

Comment:
Ive made a comment about simplifying the test to use 'EquivalentTo' where the model is mostly the same words, you could do the same here.

ProviderName has a duplicate check